pkg/relabel/relabel.go

```diff
@@ -176,6 +177,12 @@ func (re Regexp) MarshalYAML() (interface{}, error) {
 	return nil, nil
 }
 
+// MarshalJSON implements the json.Marshaler interface.
+func (re Regexp) MarshalJSON() ([]byte, error) {
+	m, err := json.Marshal(re.original)
```

Comment:
Why not return here?

Reply:
Judging by the typos in my commit messages, I'd say my brain was temporarily broken. 😖